src/components/graph-view.js

```diff
@@ -1703,7 +1698,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   }
 
   handlePanWheel = (event: any) => {
-    if (!this.props.panOnWheel || this.initialZoomInProgress) {
+    if (!this.props.panOnWheel || !this.state.viewTransform) {
```

Comment:
won't it always return once the viewTransform is set on zoom, do we ever set it to null or undefined